src/arch/riscv64/mod.rs

```diff
@@ -46,8 +50,9 @@ fn clear_bss() {
     println!("BSS cleared ({:#X} -> {:#X})", _bss_start, _bss_end);
 }
 
-/// Some architecture (x86...) have a specific instruction to write on some specific address (IO
-/// ports). RISCV don't so this is just a stub for writing at a specified address.
+/// Some architecture (x86...) have a specific instruction to write on some specific
+/// address (IO ports). RISCV does not, so this is just a stub for writing at
+/// a specified address.
```

Comment:
What is wrong with my indentation? I use rustfmt, should be good I think, no?

Reply:
Oh nothing is wrong, I simply have a 90 characters limit and it was going over it. The indentation is fine!

Reply:
Ok. I think rustfmt will break this line in the future so maybe just leave it as it is

Reply:
My bad